src/main/java/io/github/eocqrs/cmig/Master.java

```diff
@@ -54,55 +64,74 @@ public final class Master implements Scalar<String> {
   /**
    * Ctor.
    *
-   * @param doc XML
+   * @param doc XML document
    * @param cs  Cassandra
    */
   public Master(
     final XML doc,
     final Cassandra cs
   ) {
-    this.xml = doc;
+    this.self = doc;
     this.cassandra = cs;
   }
 
   /**
    * Ctor.
    *
-   * @param nm File name
-   * @param cs Cassandra
+   * @param name XML master name
+   * @param cs   Cassandra
    * @throws Exception if something went wrong
    */
   public Master(
-    final String nm,
+    final String name,
     final Cassandra cs
   ) throws Exception {
     this(
       new XMLDocument(
-        new ResourceOf(
-          "cmig/%s"
-            .formatted(nm)
-        ).stream()
+        new ResourceOf(name).stream()
       ),
       cs
     );
   }
 
+  /*
+   * @todo #48:90m/DEV Decompose Master #value().
+   */
   @Override
-  public String value() throws Exception {
-    new Names(this.xml, "").value()
-      .forEach(file -> {
-          try {
-            new InFile(
-              this.cassandra, "cmig/%s".formatted(file)
-            ).apply();
-          } catch (final Exception ex) {
-            throw new IllegalStateException(
-              "Schema migration cannot be executed",
-              ex
-            );
-          }
-        }
-      );
-    return "c8be525311cfd5f5ac7bf1c7d41a61fd82ae5e384b9b7b490358c1cb038c46c9";
+  public String value() {
+    final List<String> shas = new ListOf<>();
+    new Ids(this.self)
+      .value()
+      .forEach(id -> {
+        final String author = new Author(this.self, id)
+          .asString();
+        final String sha = new Sha(
+          id, this.self
+        ).asString();
+        new InText(
+          new TextOf(
+            "INSERT INTO cmig.states(id, author, sha, seen) "
+            + "VALUES (%s, '%s', '%s', '%s');"
+              .formatted(
```

Comment:
I'm not sure if it's safe and won't work for cql injections, wdyt?

Reply:
@h1alexbel is it possible to fix in this PR?

Reply:
@l3r8yJ only puzzle for now